packages/sftp-client/src/index.ts

```diff
@@ -112,7 +112,9 @@ export class SftpClient {
 
     log.info(`Downloading ${remoteUrl} to ${localPath}`);
 
-    await this.sftp.fastGet(remotePath, localPath);
+    await this.sftp.fastGet(remotePath, localPath, {
+      concurrency: 0,
+    });
```

Comment:
Has you tested the performance of concurrency setting?  If concurrency is 0, why do we need to call `fastGet`?